openapi-validator/build.gradle

```diff
@@ -15,9 +15,6 @@
  *
  */
 
-//apply from: "$rootDir/gradle/javaProjectWithExtBalo.gradle"
-//apply from: "$rootDir/gradle/baseNativeStdLibProject.gradle"
-
```

Comment:
Do we need to keep these commented lines?

Reply:
These are unnecessary dependencies and therefore removed from the file.